### DISCUSSION TITLE
Positive vs. Negative Sentiment Classification is 'true to the data' or 'true to the model' ?

### DISCUSSION THREAD

**Discussion by yuanjames** [shap/shap#2955] [Q&A]
Hi,

I am very curious that whether the explanation approach is 'true to the data' or 'true to the model' ? 

As the example shown in SHAP page, I can see the Partition method is used for sentimen analysis. I checked the source code that shows a default masker is set. So, I thought it is 'True to the data', I am curious that it is 'true to the data' once I gave a pretrained tokeniser and no mask?

**Comment by yuanjames** [comment]
BTW, I would like to know more details of the example, I did not see any background data, so how the explainer is trained?

**Comment by jainraj** [comment]
Can you maybe give a link to the page where you saw the example? 

As far as I know, most of the explainers are under the regime of "True to the model" i.e., they use marginal distribution instead of conditional to evaluate effect of subset of features.
